e2e/common/types.ts

```diff
@@ -6,6 +6,13 @@ export interface UserLoginInfo {
 export interface UserCredentials {
```

Comment:
```suggestion
export interface UserCredentials {
  readonly email: string;
  readonly password: string;
}
export interface UserCredentialsLong extends UserCredentials {
  readonly forename: string;
  readonly surname: string;
}
```

Could do this rather than have the duplicated email/password